build.gradle

```diff
@@ -21,11 +21,13 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.2'
+        classpath 'com.android.tools.build:gradle:8.0.2'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
-        classpath "com.vanniktech:gradle-android-junit-jacoco-plugin:0.16.0"
+        // We need to fork the plugin to fix an incorrect usage of tasks.withType(Test)
+        // https://github.com/vanniktech/gradle-android-junit-jacoco-plugin/compare/master...Laimiux:gradle-android-junit-jacoco-plugin:laimonas/gradle-8?expand=1
+        classpath("com.github.Jawnnypoo:gradle-android-junit-jacoco-plugin:61a9cc0129")
```

Comment:
Fork in order to support publishing on Jitpack. Hoping that we can see https://github.com/vanniktech/gradle-android-junit-jacoco-plugin/pull/219 get merged and won't have to worry about this anymore.

Reply:
I don't think we publish to `Jitpack` anymore

Reply:
Sorry yeah, that was confusing wording. I forked your fork of `gradle-android-junit-jacoco-plugin` so that I could change configuration of that lib to publish it to Jitpack, since it was configured to publish to IC's internal artifactory server, and was failing to build on Jitpack.